app/core/core-design-system/src/main/kotlin/com/hedvig/android/core/designsystem/material3/HedvigMaterial3Theme.kt

```diff
@@ -107,7 +107,7 @@ internal fun darkHedvigColorScheme(hedvigTonalPalette: HedvigTonalPalette) = Hed
   onContainedButtonContainer = hedvigTonalPalette.greyscale1000,
   secondaryContainedButtonContainer = hedvigTonalPalette.greyscale800,
   onSecondaryContainedButtonContainer = hedvigTonalPalette.greyscale100,
-  alwaysBlackContainer = hedvigTonalPalette.greyscale700,
+  alwaysBlackContainer = hedvigTonalPalette.greyscale900,
```

Comment:
This is now going to be greyscale900 on top of a greyscale1000 background. Is this what design wanted here? The contrast between the two is quite low is my concern, it barely looks like a card in high light environments or if the phone's brightness is not that high :D

Reply:
They said they want it the same as the expandable fields below, and the expandable fields have, if I'm not mistaken, Surface as container color - and it's this one. Actually, at the meeting William said that the expandables are too dark, but afterwards he sent me the right color, and it's #303030 - which is exactly what we had there, Surface color for dark theme

Reply:
Alrighty then, sounds good to me. Then how about we remove the other change here with the delay before we close the flow and for now do nothing there so that we can merge the design changes at least.
Then when we make the decision on if the flow will be eagerly closed or not we can add that code later. It'd be a shame for the design changes to be postponed from being merged because of the other issue

Reply:
Removed the delay with the last commit. I guess it's okay to merge now?